mtp_cashbook/templates/cashbook/email/credited-confirmation.txt

```diff
@@ -4,9 +4,9 @@ GOV.UK - {% trans 'Send money to someone in prison' %}
 
 {% trans 'Dear sender,' %}
 
-{% blocktrans trimmed %}
+{% if ref_number %}{% blocktrans trimmed %}
 The payment you made - confirmation number {{ ref_number }} - has now been credited to the prisoner’s account.
-{% endblocktrans %}
+{% endblocktrans %}{% else %}{% trans 'The payment you made has now been credited to the prisoner’s account.' %}{% endif %}
```

Comment:
Very minor stylistic comment, is it worth breaking this whole block out in to a multiline statement?

Reply:
i agree it looks gross! but otherwise you end up with a text-only email with 3+ blank lines. probably not a big deal since i'd expect very few people to see it, i was just aiming for tidier text email rather than tidier template

Reply:
alas you cannot do something like `{%- if -%}` in django templates the way you can in jinja2